shared/Microsoft.AspNetCore.RangeHelper.Sources/RangeHelper.cs

```diff
@@ -41,6 +54,8 @@ internal static class RangeHelper
             // Perf: Check for a single entry before parsing it
             if (rawRangeHeader.Count > 1 || rawRangeHeader[0].IndexOf(',') >= 0)
             {
+                logger.LogWarning("Multiple ranges are not supported.");
```

Comment:
This warning is not actionable by the server / app, it's a client issue. Debug?

Reply:
Why this one log statement? I see five other failures in this method that could be logged if we choose.

Reply:
> This warning is not actionable by the server / app, it's a client issue. Debug?

Ideally its not a client issue though, right? because the spec supports it and the user could be giving the correct ranges.

> Why this one log statement? I see five other failures in this method that could be logged if we choose.

For other failures, a user can take a look at the range header of the request and figure out the issue, but in case of multiple ranges, we would reject even if they were correct.

Reply:
>Ideally its not a client issue though, right? because the spec supports it and the user could be giving the correct ranges.

I called it a client issue because there's nothing we allow the application developer to do about this on the server, they can only change the client's behavior. 

The other failures would still warrant debug or trace logs. Any error we detect should be visible in the logs at some level.